sky/cli.py

```diff
@@ -5230,75 +5247,158 @@ def local_up():
+        gpu_hint = ''
+        if gpus:
+            # Get GPU model by querying the node labels
+            label_name_escaped = 'skypilot.co/accelerator'.replace('.', '\\.')
+            gpu_type_cmd = f'kubectl get node skypilot-control-plane -o jsonpath=\"{{.metadata.labels[\'{label_name_escaped}\']}}\"'  # pylint: disable=line-too-long
+            try:
+                # Run the command and capture the output
+                gpu_count_output = subprocess.check_output(gpu_type_cmd,
+                                                           shell=True,
+                                                           text=True)
+                gpu_type_str = gpu_count_output.strip() + ' '
+            except subprocess.CalledProcessError as e:
+                output = str(e.output.decode('utf-8'))
+                logger.warning(f'Failed to get GPU type: {output}')
+                gpu_type_str = ''
+
+            # Get number of GPUs (sum of nvidia.com/gpu resources)
+            gpu_count_command = 'kubectl get nodes -o=jsonpath=\'{range .items[*]}{.status.allocatable.nvidia\\.com/gpu}{\"\\n\"}{end}\' | awk \'{sum += $1} END {print sum}\''  # pylint: disable=line-too-long
+            try:
+                # Run the command and capture the output
+                gpu_count_output = subprocess.check_output(gpu_count_command,
+                                                           shell=True,
+                                                           text=True)
+                gpu_count = gpu_count_output.strip(
+                )  # Remove any extra whitespace
+                gpu_message = f' and {gpu_count} {gpu_type_str}GPUs'
+            except subprocess.CalledProcessError as e:
+                output = str(e.output.decode('utf-8'))
+                logger.warning(f'Failed to get GPU count: {output}')
+                gpu_message = f' with {gpu_type_str}GPU support'
+
+            gpu_hint = (
+                '\nHint: To see the list of GPUs in the cluster, '
+                'run \'sky show-gpus --cloud kubernetes\'') if gpus else ''
+
         if num_cpus < 2:
             click.echo('Warning: Local cluster has less than 2 CPUs. '
                        'This may cause issues with running tasks.')
         click.echo(
-            'Local Kubernetes cluster created successfully with '
-            f'{num_cpus} CPUs. `sky launch` can now run tasks locally.'
+            f'{style.BRIGHT}Local Kubernetes cluster created successfully with '
+            f'{num_cpus} CPUs{gpu_message}. `sky launch` can now run tasks '
+            f'locally.{style.RESET_ALL}'
             '\nHint: To change the number of CPUs, change your docker '
             'runtime settings. See https://kind.sigs.k8s.io/docs/user/quick-start/#settings-for-docker-desktop for more info.'  # pylint: disable=line-too-long
-        )
+            f'{gpu_hint}')
 
 
 @local.command('down', cls=_DocumentedCodeCommand)
 @usage_lib.entrypoint
 def local_down():
```

Comment:
Hmm, this is expected, right? Since Kubernetes cluster has been downed, there are no more clouds enabled. I was thinking if it can be presented as a warning, but will need to [refactor `check`](https://github.com/skypilot-org/skypilot/blob/a31fdd93bdfa342393a61ed603057ec5f233b2b0/sky/check.py#L55-L62) to raise errors and handle those errors upstream instead of directly writing to CLI.

Reply:
Makes sense! Thanks for explaining! The current way seems good to me.